_posts/2016-04-26-gsoc.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: GSoC Projects
+---
+
+<p>
+<img
+src="https://www.python.org/static/img/python-logo.png"
+title="Python Software Foundation" alt="Python Software Foundation"
+style="float: right; width: 10em; height: 4.5em;" />
+
+<img
+src="https://developers.google.com/open-source/gsoc/images/gsoc2016-sun-373x373.png"
+title="Google Summer of Code 2016" alt="Google Summer of Code 2016"
+style="float: left; width: 4.5em; height: 4.5em; " />
+</p>
+
+We are happy to anounce that MDAnalysis is hosting two [GSoC][gsoc] students for
+the [PSF][psf] this year, [Fiona Naughton][fiona-gsoc] and
+[John Detlefs][john-gsoc].
+
+# Fiona Naughton: Umbrella simulations with MDAnalysis
+
+<img
+src="https://avatars1.githubusercontent.com/u/10507372?v=3&s=460"
+title="Fiona Naughton" alt="Fiona Naughton"
+style="float: left; width: 110px; height: 110px; border-radius: 20px; border: 15px solid white" />
+
+[Umbrella Sampling][us] is an MD technique which involves performing a series of
+simulations in which a *reaction coordinate* , such as the distance between two
+molecules, is restrained to different values. A method to analyse the
+combination of the simulations is [weighted histogram analysis method][wham]
+(WHAM). Fiona will work on adding WHAM to the analysis module to calculate
+different observables.
+
+Fiona Naughton is doing a PhD in Biochemistry at the University of Oxford with
+the Structural Bioinformatics and Computational Biochemistry Unit, studying
+protein/membrane interactions through molecular dynamics simulations. She has
+started her own [blog][fiona-blog] and can be found in github under
```

Comment:
URL for Fiona's blog needed – @fiona-naughton can you supply a working UR, please?